algorithmic_efficiency/workloads/mnist/mnist_pytorch/workload.py

```diff
@@ -205,7 +219,7 @@ def _eval_model(
     targets = batch['targets']
     weights = batch.get('weights')
     if weights is None:
```

Comment:
Can `weights` ever be `None` here after the change above?

Reply:
`weights` will never be `None` with the above change. However, I think it might still be good to have this line for consistency and testing.